src/util/util.hpp

```diff
@@ -45,6 +46,21 @@ namespace util
     bool loadConfigFile(
         const std::filesystem::path &path, const std::function<bool(std::string_view, std::string_view)> &handleValue);
 
+    /// Generate a random number in the interval [min, max]
+    ///
+    /// @tparam numeral type of the numbers.
+    /// @param min minimum value.
+    /// @param max maximum value.
+    /// @return numeral value in the interval [min, max]
+    template <typename numeral> numeral randInt(numeral min, numeral max)
+    {
+        std::random_device rd;
```

Comment:
I don't known much about it, but insn't it kind of heavy to re-create a random engine every time we want to generate a random number?